includes/Story_Post_Type.php

```diff
@@ -254,7 +254,7 @@ public function bulk_post_updated_messages( array $bulk_messages, $bulk_counts )
 		$bulk_messages[ self::POST_TYPE_SLUG ] = [
 			/* translators: %s: Number of stories. */
 			'updated'   => _n( '%s story updated.', '%s stories updated.', $bulk_counts['updated'], 'web-stories' ),
-			'locked'    => ( 1 === $bulk_counts['locked'] ) ? __( '1 story not updated, somebody is editing it.', 'web-stories' ) :
+			'locked'    =>
```

Comment:
This is copied from core. 

https://github.com/WordPress/wordpress-develop/blob/47c9387402e4bdfe816124920186e4859c576874/src/wp-admin/edit.php#L371-L373

Why change it?

Reply:
It's superfluous.

`1 story not updated, somebody is editing it.` is equivalent to `%s story not updated, somebody is editing it.` with the placeholder replaced.

Keeping both means translators have to translate two almost identical strings for no reason.

The only situation where it would make sense to keep this is when the first version says `One story not updated, somebody is editing it.` or `Story not updated, somebody is editing it.`.

Reply:
Should this be fixed in core?

Reply:
Asking in WP Slack about background of this...

Reply:
There's a good explanation here: https://core.trac.wordpress.org/ticket/28502#comment:23

Slightly updated the wording again now to accommodate for both aspects.